prebuild_binaries/native/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian:jessie
+
+RUN apt-get update
+RUN apt-get install -y curl build-essential python libc6-dev-i386 lib32stdc++-4.9-dev
+RUN curl -fsSL get.docker.com | bash
+
+RUN mkdir /usr/local/nvm
+ENV NVM_DIR /usr/local/nvm
+
+RUN curl curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
Is there to do this without relying on dynamically downloaded components? It would be nice if the prebuilt binaries were built in a hermetic way.

Reply:
What would you propose we use to download node.js instead of nvm?

I assume that is would be possible to avoid dynamically downloading components, but that it would end up requiring a lot more maintenance, so probably not worth the effort. 

gRPC does builds this way, and that module is probably somewhat more sensitive and critical that the profiling agent.

Reply:
Can you use the [official](https://hub.docker.com/_/node/) Node.js docker image? For example
`FROM node:10-jessie` or `FROM node:8-jessie` etc.

Reply:
Thanks! 
Done!